internetarchive/cli/ia_download.py

```diff
@@ -33,6 +33,8 @@
     -R, --retries=<retries>                  Set number of retries to <retries> [default: 5].
     -I, --itemlist=<file>                    Download items from a specified file. Itemlists should
                                              be a plain text file with one identifier per line.
+    -n, --start-idx=<n>                      Start immediately at item <n>
+    -m, --end-idx=<m>                        End download after item <m>
```

Comment:
Change the long argument name to --end-index. The short from `-m` should probably be removed.